test/integration/user_commands.cc

```diff
@@ -671,3 +672,227 @@ TEST_F(UserCommandsTest, Pose)
   ASSERT_NE(nullptr, poseComp);
   EXPECT_NEAR(500.0, poseComp->Data().Pos().Y(), 0.2);
 }
+
+/////////////////////////////////////////////////
+TEST_F(UserCommandsTest, Light)
+{
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/test/worlds/lights.sdf";
+  serverConfig.SetSdfFile(sdfFile);
+
+  Server server(serverConfig);
+  EXPECT_FALSE(server.Running());
+  EXPECT_FALSE(*server.Running(0));
+
+  // Create a system just to get the ECM
+  EntityComponentManager *ecm{nullptr};
+  test::Relay testSystem;
+  testSystem.OnPreUpdate([&](const gazebo::UpdateInfo &,
+                             gazebo::EntityComponentManager &_ecm)
+      {
+        ecm = &_ecm;
+      });
+
+  server.AddSystem(testSystem.systemPtr);
+
+  // Run server and check we have the ECM
+  EXPECT_EQ(nullptr, ecm);
+  server.Run(true, 1, false);
+  EXPECT_NE(nullptr, ecm);
+
+  msgs::Light req;
+  req.set_name("point");
```

Comment:
What is this request supposed to do?

Reply:
This just add a light with the default parameters except the name and parent id